packages/payment-detection/src/eth/proxy-info-retriever.ts

```diff
@@ -86,6 +110,9 @@ export default class ProxyEthereumInfoRetriever
         parameters: {
           block: blockNumber,
           txHash: transactionHash,
+          to: this.toAddress,
+          feeAddress: parsedLog.feeAddress || undefined,
```

Comment:
```suggestion
          feeAddress: parsedLog.feeAddress
```
Should be enough if parsedLog is always defined